src/PIL/ImageShow.py

```diff
@@ -282,12 +302,11 @@ class XVViewer(UnixViewer):
     This viewer supports the ``title`` parameter.
     """
 
-    def get_command_ex(self, file, title=None, **options):
+    def get_command_ex(self, file, **options):
         # note: xv is pretty outdated.  most modern systems have
         # imagemagick's display command instead.
-        command = executable = "xv"
-        if title:
-            command += f" -name {quote(title)}"
```

Comment:
Could you explain why you think `xv` doesn't need a title argument?

Reply:
The **-name** option: sets string displayed in image window's title bar. I thought that this option is not particularly useful and removed it. Now I realize that this had an effect on documentation, including the ImageShow.show() method.